pkg/extend/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM linuxkit/alpine:488aa6f5dd2d8121a3c5c5c7a1ecf97c424b96ac AS mirror
+
+RUN mkdir -p /out/etc/apk && cp -r /etc/apk/* /out/etc/apk/
+RUN apk add --no-cache --initdb -p /out \
+    alpine-baselayout \
+    busybox \
+    e2fsprogs \
+    e2fsprogs-extra \
+    btrfs-progs \
+    xfsprogs \
+    xfsprogs-extra \
+    musl \
+    sfdisk \
+    util-linux \
+    && true
+RUN rm -rf /out/etc/apk /out/lib/apk /out/var/cache
+
+FROM linuxkit/alpine:9bcf61f605ef0ce36cc94d59b8eac307862de6e1 AS build
```

Comment:
any reason you are using two different alpine base images in the same Dockerfile?